### PR TITLE
wakuv2-test: rln-relay testnet 3 vars

### PR DESCRIPTION
**Pull request by rymnc**
This PR updates group vars for the wakuv2.test fleet, for rln-relay testnet 3 (ctx: https://github.com/waku-org/nwaku/issues/1624)

### CODE REVIEW

File: ansible/group_vars/wakuv2-test.yml [hunk 1]
```diff
@@ -50,11 +50,10 @@ nim_waku_websocket_ssl_cert: '/etc/letsencrypt/live/{{ nim_waku_websocket_domain
...
 
 # Waku rln-relay parameters
-nim_waku_rln_relay_content_topic: '/toy-chat/2/luzhou/proto'
+nim_waku_rln_relay_content_topic: '/toy-chat/3/mingde/proto'
 nim_waku_rln_relay_dynamic: true
-nim_waku_rln_relay_eth_account_address: '0000000000000000000000000000000000000000'
-nim_waku_rln_relay_eth_contract_address: '0x4252105670fE33D2947e8EaD304969849E64f2a6'
-nim_waku_rln_relay_eth_client_address: 'ws://linux-01.he-eu-hel1.nimbus.prater.wg:9547'
+nim_waku_rln_relay_eth_contract_address: '0x9C09146844C1326c2dBC41c451766C7138F88155'
+nim_waku_rln_relay_eth_client_address: 'ws://linux-01.he-eu-hel1.nimbus.sepolia:9557'
```
Comment (jakubgs) [inline ansible/group_vars/wakuv2-test.yml:56]: That's not going to fly for 3 reasons:

1. You're missing the `.wg` prefix for our WireGuard VPN addresses.
2. The port won't be open on the firewall for that host, so access from `wakuv2.test` will not work.

```
admin@node-01.do-ams3.wakuv2.test:~ % sudo nmap -Pn -p9557 linux-01.he-eu-hel1.nimbus.sepolia.wg
Starting Nmap 7.80 ( https://nmap.org ) at 2023-05-17 09:22 UTC
Nmap scan report for linux-01.he-eu-hel1.nimbus.sepolia.wg (10.14.0.58)
Host is up.

PORT     STATE    SERVICE
9557/tcp filtered unknown

Nmap done: 1 IP address (1 host up) scanned in 2.17 seconds
```
But I could open it as we did for Prater in https://github.com/status-im/infra-nim-waku/issues/59:
https://github.com/status-im/infra-nimbus/blob/c15e1ea35806a66eb1f044a5620718cc4415b0dd/ansible/group_vars/nimbus.prater.yml#L159-L160

Comment (rymnc) [reply ansible/group_vars/wakuv2-test.yml:56]: I'll add the wg prefix, could you please allow access from wakuv2.test?

**Review by jakubgs — APPROVED** [review]
I've opened the WebSocket ports:

- https://github.com/status-im/infra-nimbus/commit/597b85a0 - nimbus.sepolia: open EL WebSocket to wakuv2.test

```
admin@node-01.do-ams3.wakuv2.test:~ % sudo nmap -Pn -p9557 linux-01.he-eu-hel1.nimbus.sepolia.wg
Starting Nmap 7.80 ( https://nmap.org ) at 2023-05-17 16:17 UTC
Nmap scan report for linux-01.he-eu-hel1.nimbus.sepolia.wg (10.14.0.58)
Host is up (0.068s latency).

PORT     STATE SERVICE
9557/tcp open  unknown

Nmap done: 1 IP address (1 host up) scanned in 0.16 seconds
```
Please fix the address.

**Comment by jakubgs** [comment]
Deployed on `wakuv2.test`.

**Pull request merged**